src/material/datepicker/datepicker.ts

```diff
@@ -198,6 +206,14 @@ export class MatDatepicker<D> implements OnDestroy, CanColor {
   }
   private _disabled: boolean;
 
+  /** Position of the datepicker in the X axis. */
```

Comment:
Should this say "Preferred position", and note that it may fallback to something else if there is no space?